doc/api/http.md

```diff
@@ -648,6 +648,13 @@ stored without modification. Therefore, [`request.getHeader()`][] may return
 non-string values. However, the non-string values will be converted to strings
 for network transmission.
 
+If [`response.writeHead()`][] method is called and this method has not been
```

Comment:
This is inserted in an incorrect area or has the wrong links.

Reply:
fixed, ptal.